docs/how-to/use-trained-models.txt

```diff
@@ -0,0 +1,197 @@
+.. _use-trained-models:
+
+Using Trained Checkpoints
+=========================
+
+Determined provides APIs for downloading trained checkpoints and loading them
+into memory in a python process.
+
+The following guide will discuss:
+
+#. Querying trained model checkpoints from trials and experiments.
+#. Loading models into memory in a python process.
+#. Using the determined CLI to download checkpoints to disk.
+
+Querying Checkpoints
+--------------------
+
+The :class:`ExperimentReference <determined.experimental.ExperimentReference>`
+class is a reference to an experiment. It is retrieved via the
+:class:`Determined <determined.experimental.Determined>` class. The reference
+contains the ``top_checkpoint`` method. Without arguments, the method will
+check the experiment configuration searcher field for the ``metric`` and
+``smaller_is_better`` values. These values are used to sort the experiments
+checkpoints by validation performance. The following is a snippet from an
+experiment configuration file will result in checkpoints being sorted by the
+loss metric in ascending order.
+
+.. code:: yaml
+
+   searcher:
+     metric: "loss"
+     smaller_is_better: true
+
+The following snippet can be run with any python interpreter after the
+specified experiment has generated a checkpoint. It returns an
+instance of :class:`Checkpoint <determined.experimental.Checkpoint>`
+representing the checkpoint from an experiment with the best validation metric.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  checkpoint = Determined().get_experiment(id).top_checkpoint()
+
+Checkpoints can be sorted by any metric, by using ``sort_by``, which states
+which metric to use, and ``smaller_is_better``, which defines whether to sort
+the checkpoints in ascending or descending order with respect to the
+``sort_by`` metric.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  checkpoint = Determined().get_experiment(id).top_checkpoint(sort_by="accuracy", smaller_is_better=False)
+
+You may also query the top n checkpoints to receive a sorted list of
+checkpoints using the ``top_n_checkpoints`` method. This method also accepts
+``sort_by`` and ``smaller_is_better`` arguments.
+
+The following snippet returns the top 5 checkpoints for a specified experiment.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  checkpoints = Determined.get_experiment(id).top_n_checkpoints(5)
+
+
+The :class:`TrialReference <determined.experimental.TrialReference>` is used for
+more fine grained control over checkpoint selection. It contains a
```

Comment:
Nit: `more` feels redundant, given we're stating it's used for `fine grained control`, anyways.